### DISCUSSION TITLE
More Server Network Features?

### DISCUSSION THREAD

**Discussion by ICodeDumbStuff** [BG-Software-LLC/SuperiorSkyblock2#706] [Pending Feature Requests] [answered]
Hello,
What i am requesting is

Islands saved to a database and loaded on one of select "Island servers" and is on there when everyone leaves the island (goes offline or somthing) The island is saved to the db and removed from the said island server

Cross Server Commands - To Interact with the islands...

User Inventory Saving and sync

User Profiles???? idk just a thought on this one

EDIT: Perferably saved to all the same database for example mongodb

**Comment by Lemvs** [comment, marked as answer]
https://bg-software.com/superiorskyblock/proxybridge/

@OmerBenGera you can mark this answered and close it.
